varname.py

```diff
@@ -18,27 +17,40 @@
     "inject", "nameof", "namedtuple", "Wrapper", "debug"
 ]
 
+# To show how the desired frame is selected and other frames are skipped
+DEBUG = False
+
 class VarnameRetrievingError(Exception):
     """When failed to retrieve the varname"""
 
 def varname(
         caller: int = 1,
-        ignore: Optional[List[Union[str, Tuple[ModuleType, str]]]] = None,
+        ignore: Optional[
+            List[Union[ModuleType, Tuple[ModuleType, str]]]
+        ] = None,
         multi_vars: bool = False,
         raise_exc: bool = True
 ) -> Optional[Union[str, Tuple[Union[str, tuple]]]]:
     """Get the variable name that assigned by function/class calls
 
+    To debug and specify the right caller and ignore arguments, you can set
+    debug on and see how the frames are ignored or selected:
+
+    >>> import varname
+    >>> varname.DEBUG = True
+
     Args:
         caller: The call stack index, indicating where this function
             is called relative to where the variable is finally retrieved
-            If `ignore` provided, this should be the stack where we start
-            to search the node that should not be ignored.
-        ignore: A list of qualnames or tuples of module and qualname that
+            Stacks are counted with the ignored ones being excluded.
```

Comment:
There is one stack, consisting of many frames.

Reply:
Is frame the right word? I am sort of confused. Are we ignoring frames or stacks?

Reply:
Yes, we are ignoring and skipping frames.